src/core/schema.service.impl.ts

```diff
@@ -164,18 +189,19 @@ export class SchemaServiceImpl implements SchemaService {
         }
         const href: string = link.href;
         const variableWrapped = href.match(/\{.*\}/)[0];
-        const pathToContainment = href.split(/\{.*\}/)[0];
+        const pathToContainment = href.split(/\/\{.*\}/)[0];
         const variable = variableWrapped.substring(1, variableWrapped.length - 1);
+        const findTargets = findReferenceTargets(pathToContainment, targetSchema.id);
```

Comment:
`findTargets` is actually a function that takes the root data as a parameter. So I think this name makes more sense.